test/src/userobjects/ToggleMeshAdaptivity.C

```diff
@@ -23,17 +23,16 @@ validParams<ToggleMeshAdaptivity>()
   InputParameters params = validParams<GeneralUserObject>();
   params.addRequiredParam<MooseEnum>(
       "mesh_adaptivity", state, "Control mesh adaptivity, choices are 'on' or 'off'.");
-  params.addParam<unsigned int>(
-      "apply_after_timestep",
-      0,
-      "Number of time steps to wait before applying the adaptivity state.");
+  params.addParam<int>("apply_after_timestep",
+                       0,
+                       "Number of time steps to wait before applying the adaptivity state.");
   return params;
 }
 
 ToggleMeshAdaptivity::ToggleMeshAdaptivity(const InputParameters & params)
   : GeneralUserObject(params),
     _state(getParam<MooseEnum>("mesh_adaptivity")),
-    _steps_to_wait(getParam<unsigned int>("apply_after_timestep"))
+    _steps_to_wait(getParam<int>("apply_after_timestep"))
```

Comment:
Hmmm, so MOOSE supports negative time steps? I had the assumption in mind that you always start at 0 when I wrote this.

Reply:
I think it may, I want to say somebody takes advantage of this, but we don't really test it in MOOSE. It's definitely a signed integer whether we support it or not!